uniffi_bindgen/src/bindings/swift/gen_swift/object.rs

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::fmt;
+
+use crate::bindings::backend::{CodeDeclaration, CodeOracle, CodeType, Literal};
+use crate::interface::{ComponentInterface, Object};
+use askama::Template;
+
+use super::filters;
```

Comment:
Huh, I'm curious why clippy is okay without the `allow_unused` here 🤔

Reply:
I should imagine its been used by the `Template` macro below.